code/game/objects/structures/displaycase.dm

```diff
@@ -168,7 +168,7 @@
 		src.icon_state = "glassbox2b"
 	else
 		src.icon_state = "glassbox2[locked]"
-	overlays = 0
+	overlays.len = 0
```

Comment:
This change does nothing and I wouldn't even be surprised if this made it *slower*.